guildwars2/characters.py

```diff
@@ -411,3 +446,14 @@ def handle_duplicates(upgrades):
             except:
                 return None
         return None
+
+    def get_crafting(self, character):
+        craft_list = []
+        if character["crafting"]:
+            for crafting in character["crafting"]:
+                rating = crafting["rating"]
+                discipline = crafting["discipline"]
+                craft_list.append("Level {} {}".format(rating, discipline))
+            return craft_list
+        else:
+            return None
```

Comment:
Can be shortened a bit:
```py
def get_crafting(self, character):
    craft_list = []
    for crafting in character["crafting"]:
        rating = crafting["rating"]
        discipline = crafting["discipline"]
        craft_list.append("Level {} {}".format(rating, discipline))
    return craft_list
```
Above, you'll need to do
```py
if craft_list:
```
instead of
```py
if craft_list is not None:
```

Reply:
Changed it